notebooks/278-kosmos2-multimodal-large-language-model/278-kosmos2-multimodal-large-language-model.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.            ["https://huggingface.co/microsoft/kosmos-2-patch14-224/resolve/main/snowman.png", "An image of"],
</pre><p>does it correctly work? If I right remember before gradio required only local paths to images as examples</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1483/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>